test/token-service/test_tokenCreateTransaction.js

```diff
@@ -0,0 +1,8187 @@
+    });
+
+    it("(#10) Creates a token with an expiration time of 60 days (5,184,000 seconds) from the current time", async function () {
+      const expirationTime = parseInt(Date.now() / 1000 + 5184000);
+      const response = await JSONRPCRequest("createToken", {
+        name: "testname",
+        symbol: "testsymbol",
+        treasuryAccountId: process.env.OPERATOR_ACCOUNT_ID,
+        expirationTime: expirationTime,
+      });
+      if (response.status === "NOT_IMPLEMENTED") {
+        this.skip();
+      }
+
+      await verifyTokenCreationWithExpirationTime(
+        response.tokenId,
+        expirationTime,
+      );
+    });
+
+    it("(#11) Creates a token with an expiration time of 30 days (2,592,000 seconds) from the current time", async function () {
+      const expirationTime = parseInt(Date.now() / 1000 + 2592000);
+      const response = await JSONRPCRequest("createToken", {
+        name: "testname",
+        symbol: "testsymbol",
+        treasuryAccountId: process.env.OPERATOR_ACCOUNT_ID,
+        expirationTime: expirationTime,
+      });
+      if (response.status === "NOT_IMPLEMENTED") {
+        this.skip();
+      }
+
+      await verifyTokenCreationWithExpirationTime(
+        response.tokenId,
+        expirationTime,
+      );
+    });
+
+    //it("(#12) Creates a token with an expiration time of 30 days minus one second (2,591,999 seconds) from the current time", async function () {
+    //  try {
+    //    const response = await JSONRPCRequest("createToken", {
+    //      name: "testname",
+    //      symbol: "testsymbol",
+    //      treasuryAccountId: process.env.OPERATOR_ACCOUNT_ID,
+    //      expirationTime: (Date.now() / 1000) + 2591999
+    //    });
+    //    if (response.status === "NOT_IMPLEMENTED") this.skip();
+    //  } catch (err) {
+    //    assert.equal(err.data.status, "INVALID_EXPIRATION_TIME");
+    //    return;
+    //  }
+    //
+    //  assert.fail("Should throw an error");
+    //});
+
+    it("(#13) Creates a token with an expiration time of 8,000,001 seconds from the current time", async function () {
+      const expirationTime = parseInt(Date.now() / 1000 + 8000001);
+      const response = await JSONRPCRequest("createToken", {
+        name: "testname",
+        symbol: "testsymbol",
+        treasuryAccountId: process.env.OPERATOR_ACCOUNT_ID,
+        expirationTime: expirationTime,
+      });
+      if (response.status === "NOT_IMPLEMENTED") {
+        this.skip();
+      }
+
+      await verifyTokenCreationWithExpirationTime(
+        response.tokenId,
+        expirationTime,
+      );
+    });
+
+    it("(#14) Creates a token with an expiration time of 8,000,002 seconds from the current time", async function () {
+      try {
+        const response = await JSONRPCRequest("createToken", {
+          name: "testname",
+          symbol: "testsymbol",
+          treasuryAccountId: process.env.OPERATOR_ACCOUNT_ID,
+          expirationTime: Date.now() / 1000 + 8000002,
```

Comment:
this  number is of type `double` but it should not have floating point.

Reply:
and after rounding the number the transaction goest through, but the assert is otherwise

Reply:
We will modify how we handle the time values by converting them to ISO string format, as the current method for processing times is incorrect.

example for this specific test:
```
        const expirationDate = new Date(Date.now() + 8000002000);
        const expirationTimeIso = expirationDate.toISOString();
```
And we will send the **expirationTimeIso**

Reply:
The problem occurs when we attempt to add or remove time using numbers alone. When we create a new Date() on the server (SDK) with these numeric values, it often results in an invalid date. This invalid date is then passed to the setter, causing an '**INVALID_EXPIRATION_TIME**' error. This error leads to the test passing, which is not the expected behavior. 

On the other hand, using ISO string format for the expiration time is a more reliable approach. Since ISO strings represent a standardized date format, they can't produce incorrect or invalid date values, ensuring that the expiration time is processed correctly while sending it as a string.

Reply:
Understood, if the JS SDK is unable to parse a `BigInt` or `Long` into a `Date` then I'm okay with this move. Since we're changing the type of `expirationTime`, be sure to update the test specifications with the new type, as well as a description specifying it should be the ISO string format. I'll make an issue to change `expirationTime` parameters in other request types as well.

Reply:
#259